PsfRuntime/CreateProcessHook.cpp

```diff
@@ -24,6 +24,8 @@
 #include <psf_framework.h>
 
 #include "Config.h"
+#include <known_folders.h>

```

Comment:
Is this even needed?

Reply:
This is included for calculating appData folder path in argument redirection. Moving it to ArgumentRedirection.h file